### PR TITLE
added haldiram fastfood chain

### PR DESCRIPTION
**Pull request by Apurva-Singh**
added haldiram's fastfood chain

### CODE REVIEW

File: data/brands/amenity/fast_food.json [hunk 1]
```diff
@@ -1728,6 +1728,21 @@
...
       }
     },
+    {
+      "displayName": "Haldiram's",
+      "id": "haldirams-aba431",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "alt_name": "Haldiram",
+        "amenity": "fast_food",
+        "brand": "Goli Vada Pav",
```
Comment (bhousel) [inline data/brands/amenity/fast_food.json:1738]: Don't forget to change this tag too..
```suggestion
        "brand": "Haldiram's",
```

**Review by bhousel — CHANGES_REQUESTED** [review]
Thanks @Apurva-Singh - mostly good but looks like one tag was missed?

File: data/brands/amenity/fast_food.json (hunk 1 above)
Comment (Apurva-Singh) [reply data/brands/amenity/fast_food.json]: Hey @bhousel I have made the requested changes, please review.

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
Thanks @Apurva-Singh & welcome to the project 👋

**Pull request merged**